src/accountlist/accountlistReducers.js

```diff
@@ -0,0 +1,57 @@
+import { fromJS } from 'immutable';
+
+import {
+  REALM_ADD,
+  LOGIN_SUCCESS,
+  LOGOUT,
+  ACCOUNT_REMOVE,
+} from '../account/accountActions';
+
+const initialState = fromJS([]);
+
+export default (state = initialState, action) => {
```

Comment:
Let's reconsider this after we discuss with Tim and others if this behavior is ideal, and what we might want to add to the login flow (maybe not much). I definitely prefer the array, if no timestamp is ever used, as the state is not a db and using the array's property to be ordered is a valid approach, IMHO.